react/src/components/atoms/forms/Form/index.js

```diff
@@ -24,9 +23,46 @@ class FormProvider extends Component {
       hasId: this.hasId,
       setValue: this.setValue,
       updateState: this.updateState,
-      getValues: this.getValues
+      getValues: this.getValues,
```

Comment:
I wonder if these methods need to be in the state or can directly be passed to Provider value?

Reply:
e.g.    
```suggestion
      this.methods = {
      getValue: this.getValue,
      hasId: this.hasId,
      setValue: this.setValue,
      updateState: this.updateState,
      getValues: this.getValues,
      syncContent: this.syncContent,
      getLinkedContent: this.getLinkedContent,
      updateLinkedContent: this.updateLinkedContent,
      setLinkedContent: this.setLinkedContent,
      onUpdate: this.onUpdate,
      getUpdateFunc: this.getUpdateFunc,
      setUpdateFunc: this.setUpdateFunc
    }
```